homeassistant/components/wyoming/conversation.py

```diff
@@ -0,0 +1,207 @@
+"""Support for Wyoming intent recognition services."""
+
+import logging
+
+from wyoming.asr import Transcript
+from wyoming.client import AsyncTcpClient
+from wyoming.handle import Handled, NotHandled
+from wyoming.info import HandleProgram, IntentProgram
+from wyoming.intent import Intent, NotRecognized
+
+from homeassistant.components import conversation
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers import intent
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.util import ulid
+
+from .const import DOMAIN
+from .data import WyomingService
+from .error import WyomingError
+from .models import DomainDataItem
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup_entry(
+    hass: HomeAssistant,
+    config_entry: ConfigEntry,
+    async_add_entities: AddEntitiesCallback,
+) -> None:
+    """Set up Wyoming conversation."""
+    item: DomainDataItem = hass.data[DOMAIN][config_entry.entry_id]
+    async_add_entities(
+        [
+            WyomingConversationEntity(config_entry, item.service),
+        ]
```

Comment:
Wouldn't this always add a conversation entity, even if the config entry doesn't support it?